src/vse_sync_pp/parsers/parser.py

```diff
@@ -83,21 +92,23 @@ def parse(self, file, relative=False):
             parsed = self.parse_line(line) # pylint: disable=assignment-from-none
             if parsed is not None:
                 if relative:
-                    timestamp = getattr(parsed, 'timestamp', None)
-                    if timestamp is not None:
-                        if tzero is None:
-                            tzero = timestamp
-                        parsed = parsed._replace(timestamp=timestamp - tzero)
-                yield parsed
-    def canonical(self, file):
+                    tzero, parsed = relative_timestamp(parsed, tzero)
+            yield parsed
```

Comment:
In both cases this needs to be indented by one level to allow the same behaviour as before

Reply:
fixed.